### ISSUE TITLE
Incoming messages from unknown number

### ISSUE THREAD

**Issue by hubert3** [WhisperSystems/Signal-iOS#656]
I received a text from a number not associated with any of my contacts (+61xxxxx)

A few issues with this:

-There is no easy way to create a new iPhone contact based on this number (what I'd ideally like to do)
-Seems impossible to even copy the number to the clipboard
-The circle avatar for the person is just '+'

**Comment by FredericJacobs** [comment]
@hubert3 Tapping the number should let you create a new contact. Isn't that working?

**Comment by hubert3** [comment]
@FredericJacobs Oops, yes it did work. I thought I tried that before but I must not have.

However, Signal crashed after I entered a first & last name and created the contact.

The contact was successfully created and now appears in Contacts and Signal.

```
Mar  4 18:40:48 iPhone-6 Signal[3434] <Warning>: Unable to simultaneously satisfy constraints.
		Probably at least one of the constraints in the following list is one you don't want. Try this: (1) look at each constraint and try to figure out which you don't expect; (2) find the code that added the unwanted constraint or constraints and fix it. (Note: If you're seeing NSAutoresizingMaskLayoutConstraints that you don't understand, refer to the documentation for the UIView property translatesAutoresizingMaskIntoConstraints) 
	(
	    "<NSAutoresizingMaskLayoutConstraint:0x170488340 h=--& v=--& V:[ABContactHeaderDisplayView:0x1456e3b50(47.5)]>",
	    "<ABBaselineBoundLayoutConstraint:0x1702ea780 V:[UILabel:0x1456c2ad0'xxx']-(>=24)-|   (Names: '|':ABContactHeaderDisplayView:0x1456e3b50 )>",
	    "<ABBaselineBoundLayoutConstraint:0x1702ea800 UILabel:0x1456c2ad0'xxx'.firstBaseline == ABContactHeaderDisplayView:0x1456e3b50.top + 40>"
	)
	
	Will attempt to recover by breaking constraint 
	<ABBaselineBoundLayoutConstraint:0x1702ea800 UILabel:0x1456c2ad0'xxx'.firstBaseline == ABContactHeaderDisplayView:0x1456e3b50.top + 40>
	
	Make a symbolic breakpoint at UIViewAlertForUnsatisfiableConstraints to catch this in the debugger.
	The methods in the UIConstraintBasedLayoutDebugging category on UIView listed in <UIKit/UIView.h> may also be helpful.


```

**Comment by hubert3** [comment]
Crash log uploaded at https://app.box.com/s/9ls6zouvxj3edeiebe9f7ipuqyw64iwx

**Comment by FredericJacobs** [comment]
That crash was already fixed in 2.0.1 too ;)
